packages/block-library/src/term-template/index.php

```diff
@@ -39,13 +40,18 @@ function render_block_core_term_template( $attributes, $content, $block ) {
 		'exclude'    => $query['exclude'] ?? array(),
 	);
 
-	// Handle parent.
-	if ( ! empty( $query['hierarchical'] ) && isset( $query['parent'] ) ) {
-		$query_args['parent'] = $query['parent'];
-	} elseif ( ! empty( $query['hierarchical'] ) ) {
+	// We set parent to 0 only if we show all terms as hierarchical or we show top-level terms.
+	if ( ('all' === $terms_to_show && ! empty( $query['hierarchical'] ) ) || 'top-level' === $terms_to_show ) {
```

Comment:
I think this is what the linter is complaining about:

```suggestion
	if ( ( 'all' === $terms_to_show && ! empty( $query['hierarchical'] ) ) || 'top-level' === $terms_to_show ) {
```

Reply:
OMG, sorry about that 😬 Hopefully now CI will pass 🤞